importers/nmdb-import-cisco/Parser.cpp

```diff
@@ -417,6 +494,139 @@ Parser::addVlan(nmco::Vlan& vlan)
   d.vlans.push_back(vlan);
 }
 
+
+// Policy Related
+void
+Parser::createAccessGroup(nmco::InterfaceNetwork* iface,
+                          const std::string& bookName,
+                          const std::string& direction)
+{
+  usedRuleBooks[bookName] = {iface->getName(), direction};
+}
+
+void
+Parser::updateCurrentRuleBook(const std::string& name)
+{
+  curRuleBook = name;
+  curRuleId = 0;
+}
+
+void
+Parser::updateCurrentRule()
+{
+  ++curRuleId;
+  d.ruleBooks[curRuleBook][curRuleId].setRuleDescription(curRuleBook);
+}
```

Comment:
Would it make sense to have a global reference / pointer to `d.ruleBooks[curRuleBook][curRuleId]` so that we don't have to look it up in each successive function call?

Reply:
Decided against this because none of the other parsers do this and I want to stay consistent.